files/src/utils/test-helpers/test-helper.ts

```diff
@@ -0,0 +1,7 @@
+import { setApp, start } from '@glimmer/test-helpers';
+import App from '../../main';
+
+QUnit.config.autostart = false;
```

Comment:
How would we name that? `ember-cli-qunit` is technically correct, as it hooks into ember-cli, but it currently contains ember-specific stuff.